dspace/config/registries/dublin-core-types.xml

```diff
@@ -260,6 +260,18 @@
     <qualifier>other</qualifier>
     <scope_note>A known identifier type common to a local collection.</scope_note>
   </dc-type>
+  
+  <dc-type>
+    <schema>dc</schema>
+    <element>identifier</element>
+    <qualifier>doi</qualifier>
+  </dc-type>
+  
+  <dc-type>
+    <schema>dc</schema>
+    <element>identifier</element>
+    <qualifier>scopus</qualifier>
+  </dc-type>
```

Comment:
I know these two new fields are pretty straightforward, but they both should still have a `<scope_note>`.  Notice how every other field in this file has a scope note.  These should have scope notes too.